robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -224,22 +248,18 @@ def add_batch_table_columns(self,*args):
         verify_field=npsp_lex_locators["gift_entry"]["duellist"].format("Available Fields",args[position])
         print (f'verify locator is {verify_field}')
         self.selenium.wait_until_page_does_not_contain_element(verify_field)
-
-
-            
-
-
-
-    
+  
 
 @pageobject("Form", "Gift Entry")
 class GiftEntryFormPage(BaseNPSPPage, BasePage):
 
-    def _is_current_page(self):
-        """
-        Verifies that current page is Gift Entry form page
-        """
-        self.selenium.wait_until_page_contains("Donor Information")
+    def _is_current_page(self,title=None):
+        """Verifies that current page is Gift Entry form page by making sure page contains given title if mentioned, 
+        if not then title as Donor Information"""
+        if title is not None:
+            self.selenium.wait_until_page_contains(title)
+        else:
+            self.selenium.wait_until_page_contains("Donor Information")
```

Comment:
How does this sound?
```
if title is not None:
    title = "Donor Information"
self.selenium.wait_until_page_contains(title)
```

It's obviously the same code, just a different logic. Choose as per your preference :)